contracts/EpochTokenLocker.sol

```diff
@@ -212,7 +212,8 @@ contract EpochTokenLocker {
     }
 
     function updateDepositsBalance(address user, address token) private {
-        if (balanceStates[user][token].pendingDeposits.batchId < getCurrentBatchId()) {
+        uint256 batchId = balanceStates[user][token].pendingDeposits.batchId;
```

Comment:
Maybe a comment that we do this to save an SSTORE in case there is no pending deposit would be appropriate?